containers/CSharp/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,74 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/dotnet
+{
+    "name": "C# (.NET)",
```

Comment:
>C# (.NET) [](http://example.com/codeflow?start=13&length=9)

This is not a general purpose C# dev container, so we should call it Azure IoT Edge C# or something similar